src/public/style.css

```diff
@@ -2,13 +2,14 @@ body {
   background-color: #d9e6fc;
 }
 h1 {
+  display: inline;
   font-size: 58px;
   background-color: #21e342;
-  text-align: center;
+  float: center;
```

Comment:
Also not necessary (or valid, cannot float center). Hold off on styling until you work on #8.